tests/home.spec.js

```diff
@@ -49,3 +51,23 @@ test.describe("accessibility tests (dark)", () => {
     expect(accessibilityScanResults.violations).toEqual([]);
   });
 });
+
+test.describe("Data rendered correctly on home page", () => {
+  test("Total active rendered", async ({ page }) => {
+    const { stats: totalStats } = await getTotalStats();
+    const abbreviatedActive = abbreviateNumber(totalStats.active);
+    await page.goto("/");
+    const activeLocator = await page.locator("dl > div:nth-child(1) dd div:nth-child(1)");
+    const finalTotalActive = await activeLocator.textContent();
+    expect(finalTotalActive).toEqual(abbreviatedActive);
```

Comment:
I think using the card text might be a better selector

```suggestion
    const section = page.getByText("Active Users");
    const parent = section.locator("..");
    const item = parent.locator("dd div:nth-child(1)");
    await expect(item).toHaveText(abbreviatedActive);
```